cypress/commons/actions/brewery/BreweryParameters.js

```diff
@@ -231,6 +231,14 @@ function getCustomersErrorLoc(errorIndex) {
   return TableParameters.getErrorLoc(getCustomersTable(), errorIndex);
 }
 
+function getCustomersColumnsGroup(columnsGroupIndex) {
+  return TableParameters.getColumnsGroup(getCustomersTable(), columnsGroupIndex);
+}
+
+function openCutomersColumnGroup(columnsGroupIndex) {
```

Comment:
Typo : `Cutomers`

Reply:
Also, column**s**Group in `openCutomersColumnGroup` and `openColumnGroup` to be consistent

Reply:
DONE